hkmc2/shared/src/test/mlscript/apps/IterTest.mls

```diff
@@ -11,7 +11,7 @@ open Stack { Cons, Nil }
   filtering of x => x % 2 is 0
   mapping of x => x * 2
   toArray()
-//│ = [4, 8, 12, 16, 20]
+//│ = [ 4, 8, 12, 16, 20 ]
```

Comment:
Do we really want these spaces?

Reply:
Serialization (e.g., `JSON.stringify`) should have no spaces to save space. The purpose of the `Rendering` module is pretty printing, so it should make some changes to improve readability.